### ISSUE TITLE
thunderbird looses keyboard focus when launched in birdtray on archlinux + mate + compiz

### ISSUE THREAD

**Issue by marqsbla** [gyunaev/birdtray#553]
Hi, thanks for great software,

**OS**: Arch linux, newest packages on this day. Arch Linux, kernel 6.3.4-arch2-1 (no Arch release, since rolling OS) 
**Linux Desktop Manager**: Mate (1.26.1-1) + Compiz (0.9.14.2-2) 
**Birdtray version**: 1.9.0-1
**Thunderbird version**: 102.11.2-1
**Birdtray origin**: aur package (build from source) 
**Qt version**: 5.15.9+kde+r154-1 

**Description**

I changed recently window manager back from gnome 3 to mate + compiz. Whenever I run thunderbird in birdtray, I have problem with keyboard focus being lost from thunderbird main window, after I switched to another window. When I get back to thunderbird, I need to hide and unhide to regain keyboard focus. It is pretty annoying, and I don't know where to seek for the problem. 

**Expected behavior**

Keyboard focus should be followed with window focus. 

**To Reproduce**

Most common scenario:
1.  I unhide the thunderbird window from the tray
2. I go to another window
3. I go back to thunderbird, using the mouse
4. everything within the window is working fine, except keyboard. Keyboard focus stays with the former window. 
5. After hind -> unhiding the window, the keyboard focus is restored, and I can use keyboard within thunderbird  main window (like quick search, or shortcuts)
6. The problem do not occur when running thunderbird without birdtray
7. The problem do not occur in other thunderbird windows (like new message)

**Comment by marqsbla** [comment]
I noticed that thunderbird did not appear in alt+tab application switcher. I followed this: https://wiki.archlinux.org/title/Compiz#Windows_lose_focus_when_unminimised and thunderbird seems to work properly now :). Sorry for the unrelated issue.
